src/include/executor/executor.h

```diff
@@ -113,7 +113,7 @@ extern bool ExecMaterializesOutput(NodeTag plantype);
 /*
  * prototypes from functions in execCurrent.c
  */
```

Comment:
Add comments the meaning of return true  or false.

Reply:
I add one comment at the definition site.